vendor/vendor.json

```diff
@@ -72,6 +72,90 @@
 			"version": "v0.40.0",
 			"versionExact": "v0.40.0"
 		},
+		{
```

Comment:
Quite a few new dependencies. Could you do a quick check on the binary size change?

Reply:
@ruflin , @jsoriano , same here, ~5MB, is this a reasonable size?

Reply:
I'd say so, yes.

Reply:
I guess so. But binary size is really something we should start tracking.